app/src/pages/inside/filtersPage/filterTable/filterTable.jsx

```diff
@@ -88,16 +88,19 @@ export class FilterTable extends PureComponent {
             <NoFiltersBlock />
           )}
         </div>
-        <div className={cx('filter-table-pagination')}>
-          <PaginationToolbar
-            activePage={activePage}
-            itemCount={itemCount}
-            pageCount={pageCount}
-            pageSize={pageSize}
-            onChangePage={onChangePage}
-            onChangePageSize={onChangePageSize}
-          />
-        </div>
+        {data &&
+          !!data.length && (
```

Comment:
!! unnecessary here

Reply:
So the same structure exists in the line before in this component
Can I replace it there with data.length too?
![image](https://user-images.githubusercontent.com/31861805/41399013-d8fb18b6-6fc1-11e8-98ea-ac05b40b9cc4.png)

Reply:
yeah

Reply:
hmm, let it be. Sorry for my fallacy